app/models/manageiq/providers/openshift/container_manager/refresh_worker/runner.rb

```diff
@@ -1,2 +1,21 @@
 class ManageIQ::Providers::Openshift::ContainerManager::RefreshWorker::Runner < ManageIQ::Providers::BaseManager::RefreshWorker::Runner
+  include ManageIQ::Providers::Kubernetes::ContainerManager::StreamingRefreshMixin
+
+  def connection_for_entity(entity_type)
```

Comment:
do we use these somewhere?

This was for old refresh, that fetches stuff in array. But in collector, we specify client for each entity